src/fairseq2/assets/cards/wav2vec2.yaml

```diff
@@ -12,6 +12,14 @@ checkpoint: "https://dl.fbaipublicfiles.com/fairseq/wav2vec/wav2vec2_fs2_small.p
 
 ---
 
+name: dinosr_base
+model_type: wav2vec2  # compat
+model_family: wav2vec2
+model_arch: pseudo_dinosr_base
+checkpoint: "/fsx-ust/andyyuan/pt_models/xlsr_ckpts/pseudo_dinosr_spin_ft_use7_frz5_cl4096.pt"
```

Comment:
I wonder if it's actually better to have this config, which is likely to only be tentative, only exists in seamless_next rather than the public fairseq2 repo.